packages/worker/src/workflow/bundler.ts

```diff
@@ -120,12 +120,14 @@ export class WorkflowCodeBundler {
     }
 
     // Cast because the type definitions are inaccurate
-    ufs.use(memfs.createFsFromVolume(vol) as any).use({ ...realFS, readdir: readdir as any });
+    const memoryFs = memfs.createFsFromVolume(vol);
+    ufs.use(memoryFs as any).use({ ...realFS, readdir: readdir as any });
     const distDir = '/dist';
     const entrypointPath = this.makeEntrypointPath(ufs, this.workflowsPath);
 
     this.genEntrypoint(vol, entrypointPath);
-    await this.bundle(ufs, entrypointPath, distDir);
+    await this.bundle(ufs, memoryFs, entrypointPath, distDir);
```

Comment:
@bergundy tests have passed 4 times in a row after adding this change—telling webpack to use `memoryFs` instead of `ufs` as `outputFileSystem`. That sound right? If yes, I'll remove logs and DEBUG level/tracing and merge.

Reply:
I’m wondering why that fixes the problem and why it only happens on Windows.
What happens if you list /dist? Does that directory exist in the real FS?
Anyways the fix seems fine so I’m approving

Reply:
I don't know why, it was most related thing I could think of. 

Tried stating, ENOENT in all tests:

<img width="450" alt="image" src="https://user-images.githubusercontent.com/251288/160488457-69f937f7-7dcb-4330-98cc-623935a97093.png">

[build-and-test (16, windows-latest)](https://github.com/temporalio/sdk-typescript/runs/5726434268?check_suite_focus=true#logs)

[2cb3ece](https://github.com/temporalio/sdk-typescript/pull/554/commits/2cb3ecec16c1deffac60133014fe398c0a900905)

Will add a comment linking to this PR, revert logs, and merge.

Reply:
We don't have to keep investigating, don't know how much we're getting from finding out the root cause.

One thing that could be *nice* to verify is whether webpack writes to the output to the real file system in the failing tests.

Reply:
>What happens if you list /dist? Does that directory exist in the real FS?

FYI: One of our developper who works on Windows got this issue this morning (a kitchen sink project that had no been updated to latest releases). Yes, he ended up with a `c:\dist\main.js` file on his machine, and once that happened, `bundleWorkflowCode()` would always return the content of that (stall) file instead of the actual result of bundling. Had to delete that file from his disk to get bundling to work properly again.

Reply:
Interesting, can you confirm that with newer SDK version presence of `c:\dist\main.js` doesn't affect bundling?

Reply:
See #619.